pkgs/by-name/ka/kamal-proxy/package.nix

```
@@ -0,0 +1,38 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
```

```suggestion
buildGoModule (finalAttrs: {
```

`finalAttrs` is usually preferred over `rec` where supported as it [better allows overriding](https://nixos.org/manual/nixpkgs/unstable/#mkderivation-recursive-attributes)